@shared/helpers/stellar.ts

```diff
@@ -23,32 +24,13 @@ export const MAINNET_NETWORK_DETAILS = {
 } as NetworkDetails;
 
 export const TESTNET_NETWORK_DETAILS = {
+  isTestnet: true,
```

Comment:
add these here so we can also statically import these complete configs if we need to